backend/services/deployments/config/config.go

```diff
@@ -151,6 +151,10 @@ const (
 	// migrations on the artifacts and releases collections.
 	SettingDisableNewReleasesFeature        = "disable_new_releases_feature"
 	SettingDisableNewReleasesFeatureDefault = false
+
+	// Max Request body size
+	SettingMaxRequestSize        = "request_size"
```

Comment:
Maybe we can rename the config key to something a little more self documenting?
```suggestion
	SettingMaxRequestSize        = "request_size_limit"
```

Reply:
Speaking of documentation, could you also add documentation (as comments) to the `config.yaml` too?
This is easier to refer to as it's also shipped with the container image.